app/components/summary_card/view.rb

```diff
@@ -0,0 +1,40 @@
+class SummaryCard::View < ViewComponent::Base
+  # validates :rows, presence: true
+
+  def initialize(title:, heading_level: 2, rows:, border: true, editable: true)
+    # rows = transform_hash(rows) if rows.is_a?(Hash)
```

Comment:
this needed?

Reply:
Refactored the component to only include what we have and need currently